CRM/Contribute/BAO/Contribution.php

```diff
@@ -3821,7 +3780,6 @@ public static function completeOrder($input, $recurringContributionID, $contribu
       'contribution_status_id',
       'card_type_id',
       'pan_truncation',
-      'financial_type_id',
```

Comment:
this was sort of being permitted through for `repeattransaction` (except it accesses if from `input` anyway) - it's not a pass through param for `completetransaction`